### PR TITLE
Add OWF attribution and explain !sd

### LINKED ISSUE #269: Intellectual Property concern wrt. SD-JWT examples
@bc-pi wrote (on the [VCWG mailing list](https://lists.w3.org/Archives/Public/public-vc-wg/2024May/0003.html):
> Are there any concerns from the W3C or the Verifiable Credentials Working Group regarding the use of some of the YAML constructs from the OpenWallet Foundation’s SD-JWT Reference Implementation in the 'Securing Verifiable Credentials using JOSE and COSE' publication? Specifically, these constructs appear in two examples* without attribution, explanation or reference. Could this raise issues related to publication process/procedure, intellectual property rights, or document clarity?
> \* these two examples:
https://www.w3.org/TR/vc-jose-cose/#example-a-simple-example-of-a-verifiable-credential-secured-with-sd-jwt
https://www.w3.org/TR/vc-jose-cose/#example-a-simple-example-of-a-verifiable-presentation-secured-with-sd-jwt

### PR DESCRIPTION
**Pull request by selfissued**
Fixes #269 

Cc @danielfett

***
<a href="https://pr-preview.s3.amazonaws.com/selfissued/vc-jose-cose/pull/270.html" title="Last updated on May 7, 2024, 6:38 PM UTC (f5fa731)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/270/5ba3d8d...selfissued:f5fa731.html" title="Last updated on May 7, 2024, 6:38 PM UTC (f5fa731)">Diff</a>

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Comment by iherman** [comment]
I guess the changes are, from a content point of view, fine. 

That being said, I find it a strange choice to create an example that way. The role of the spec includes serving as a document that makes it possible for newcomers to understand what the technical content is. Examples play an essential role in this. Nowhere else in the whole family of specification do we use YAML, and even nontrivial aspects thereof, which is odd.  If it is *necessary* from the SD-JWT point of view, then be it. If it is just because we use a particular tool to generate the example, I question the wisdom of the choice. I would prefer, if necessary, to manually make the example valid in JSON for readability's sake.

This is not a formal objection to the PR, just food for thoughts.

File: index.html [hunk 1]
```diff
@@ -501,6 +507,13 @@ <h2>Securing JSON-LD Verifiable Presentations with SD-JWT</h2>
           Credentials in verifiable presentations MUST be secured.
 	  These credentials are secured using SD-JWT in this case.
```
Comment (TallTed) [inline index.html:508]:
```suggestion
	  In this case, the credentials are secured using SD-JWT.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Rephrase a sentence, for clarity...

**Comment by bc-pi** [comment]
> If it is necessary from the SD-JWT point of view

It is absolutely not necessary and as mentioned in https://github.com/w3c/vc-jose-cose/issues/269#issuecomment-2094979662 I think it's inappropriate for use directly in a specification (for reasons you mentioned among others). I'd note that the SD-JWT draft itself uses the tooling to produce (hopefully useful) examples like [this one](https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-08.html#section-6) but does not include any YAML in the specification itself. But if the WG and editors of vc-jose-cose think it's okay, I'm not going to spend any more energy on the topic.

**Comment by decentralgabe** [comment]
thanks for the reference @bc-pi - I think us following what the SD-JWT spec itself does makes sense
these are just examples, and should be broadly useful/readable. if we're failing to do that, or if there are other concerns (like IPR raised earlier) then it seems like we should change tacks.

**Comment by bc-pi** [comment]
> [...] following what the SD-JWT spec itself does makes sense [...]

Pulling on that thread a little bit, I'll note that the SD-JWT draft has an example that tried to envision how SD-JWT might be used to express/secure a VCDM payload and a presentation of such with a subset of that payload disclosed. That's at:

 https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-08.html#appendix-A.4

There may be some opportunity to borrow content/concepts from that and/or unveil issues in either document. 

The presentation part in particular is rather different than whats in vc-xose's [Securing JSON-LD Verifiable Presentations with SD-JWT](https://www.w3.org/TR/2024/CRD-vc-jose-cose-20240425/#securing-json-ld-verifiable-presentations-with-sd-jwt). And, to be honest, securing a VP with SD-JWT doesn't really make sense to me. But maybe I'm not seeing the grand vision or something.

**Comment by decentralgabe** [comment]
This should be closed in favor of #272

**Comment by bc-pi** [comment]
> This should be closed in favor of #272

That PR does take the`!sd` YAML out but it introduces more issues than I'm able to identify or articulate. Maybe it'd be better to just go with the current content of this PR and defer more significant changes to https://github.com/w3c/vc-jose-cose/issues/264. And also defer addressing other things like the the entire concept of securing a VP with SD-JWT being nonsensical and/or other issues that maybe/hopefully will be surfaced by doing that work.

**Comment by bc-pi** [comment]
> > This should be closed in favor of #272
> 
> That PR does take the`!sd` YAML out but it introduces more issues than I'm able to identify or articulate. Maybe it'd be better to just go with the current content of this PR and defer more significant changes to #264. And also defer addressing other things like the the entire concept of securing a VP with SD-JWT being nonsensical and/or other issues that maybe/hopefully will be surfaced by doing that work.

#273 is different but does not fundamentally change the position stated.

**Comment by decentralgabe** [comment]
superseded by #273

**Pull request closed without merging**